hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml

```diff
@@ -49,6 +49,37 @@
         <field name="view_mode">pivot,graph</field>
     </record>
 
+    <record id="hr_attendance_theoretical_view_employee_pivot" model="ir.ui.view">
```

Comment:
Why do you need an extra view? This one is the same as `hr_attendance_theoretical_view_pivot`

Reply:
Its not the same view - the new view does have the employee_id in the col - and the date in the row. So you get a nice single employee view. This is not possible by using the same view.

Reply:
Yeah, and I don't think that is a good change... I prefer consistency in the way of showing data always in the same layout.

Reply:
The current layout is perfect when you look at more than 1 employee. But as soon as you have only 1 employee - and you have the complete history enabled (instead of the last two months) - then the horizontal view will become a pain for users. The vertical layout is much better suited for this - and is also what user do expect here. imho

Reply:
OK, let's see what others think about this, or if we maybe switch both views with this logic

Reply:
I change axis when get 1-3 Employees data. But its true that with a lot of them the original layout is better.

Reply:
Having the direct button for flipping axis makes no pain to have the same layout for both for avoiding extra XML data

Reply:
A direct button for flipping axis ? How to get this button ?

Reply:
![Peek 06-11-2019 16-25](https://user-images.githubusercontent.com/7165771/68311633-1a153e80-00b2-11ea-9604-46f020b934f7.gif)

Reply:
working now around 6 years with odoo - and have never used that button until now ;-)